### ISSUE TITLE
As product owner, I want the API discovery user interface used to find NLX services to be accessible to all citizens and to be compliant with the European "Accessibility requirements suitable for public procurement of ICT products and services in Europe" standard (EN 301 549 v1.1.2)

### ISSUE THREAD

**Issue by arnoudquanjer** [VNG-Realisatie/nlx#64]

**Comment by ehotting** [comment]
@arnoudquanjer As we will dive into UX for UI's soon, can you dig into the requirements for user interfaces related to those accessibility standards?

**Comment by arnoudquanjer** [comment]
## Web Content Accessibility Guidelines

### Requirements
Standard - EN 301 549 v1.1.2: "Accessibility requirements suitable for public procurement of ICT products and services in Europe" recommends functional accessibility requirements applicable to ICT products and services, together with a description of the test procedures and evaluation methodology for each accessibility requirement in a form that is suitable for use in public procurement within Europe. The recommendation made in this standard is that web content conforms to the Web Content Accessibility Guidelines (WCAG) 2.x specification.

After the 'Wet Digitale Overheid' and 'AMvB: Besluit digitale toegankelijkheid') are introduced in 2018 (at the latest 23 September 2018) WCAG 2.x compliance becomes mandatory: 
•	On 23 September 2019 for websites published from 23 September 2018;
•	On 23 September 2020 for websites published from before 23 September 2018;
•	On 23 June 2021 for mobile applications.

Web-pages produced as a part of NLX, such as the user interface for the NLX API directory and the NLX certificate portal, will have to comply with the specifications detailed in the WCAG 2.x specification which can be found at: https://www.w3.org/TR/WCAG20/

### WCAG 2.0 Conformance
There are five requirements that must be met for content to be classified as 'conforming' to WCAG 2.0. In short, these requirements are:

1.	The first requirement deals with the levels of conformance. It basically says that all information on a page conforms or has a conforming alternate version that is available from the page.
2.	Conformance (and conformance level) is for full Web page(s) only, and cannot be achieved if part of a Web page is excluded.
3.	When a Web page is one of a series of Web pages presenting a process (i.e., a sequence of steps that need to be completed in order to accomplish an activity), all Web pages in the process conform at the specified level or better. Conformance is not possible at a particular level if any page in the process does not conform at that level or better.
4.	Only accessibility-supported ways of using technologies are relied upon to satisfy the success criteria. Any information or functionality that is provided in a way that is not accessibility supported is also available in a way that is accessibility supported.
5.	If technologies are used in a way that is not accessibility supported, or if they are used in a non-conforming way, then they do not block the ability of users to access the rest of the page.

The conformance requirements can be found at: https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html

**Comment by FabianBui** [comment]
@arnoudquanjer I'm creating the front-end so I will take care of the accessibility requirements :)
